core/engine/src/builtins/atomics/mod.rs

```diff
@@ -55,8 +55,12 @@ impl IntrinsicObject for Atomics {
             .static_method(Atomics::sub, js_string!("sub"), 3)
             .static_method(Atomics::wait, js_string!("wait"), 4)
             .static_method(Atomics::notify, js_string!("notify"), 3)
-            .static_method(Atomics::bit_xor, js_string!("xor"), 3)
-            .build();
+            .static_method(Atomics::bit_xor, js_string!("xor"), 3);
+
+        #[cfg(feature = "experimental")]
+        let builder = builder.static_method(Atomics::pause, js_string!("pause"), 0);
```

Comment:
Should the length be 1 if it takes an iteration number?

Reply:
It's optional, so the required number of arguments is zero.